lib/hydra/works/services/generic_file/persist_derivatives_output_file.rb

```diff
@@ -0,0 +1,26 @@
+require 'hydra/derivatives'
+
+module Hydra::Works
+  class PersistDerivativesOutputFile < Hydra::Derivatives::PersistOutputFileService
```

Comment:
Though I wouldn't hold up the PR for this alone, I might like to see the class (and file) renamed more succinctly: `PersistDerivative`. That also reflects the description of the class on line 7 of this file.